pom.xml

```diff
@@ -330,7 +330,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-war-plugin</artifactId>
-        <version>2.3</version>
+        <version>3.3.2</version>

```

Comment:
I couldn't get things to run for me on my Macbook without upgrading this dependency.